carml-engine/src/test/resources/RmlMapper/nullValues/nullValueMultiValuedTemplate.rml.ttl

```diff
@@ -0,0 +1,23 @@
+@prefix rr: <http://www.w3.org/ns/r2rml#>.
+@prefix rml: <http://semweb.mmlab.be/ns/rml#>.
+@prefix ex: <http://example.com/>.
+@prefix ql: <http://semweb.mmlab.be/ns/ql#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<#TermTypeMappingLiteral> a rr:TriplesMap ;
+  rml:logicalSource [
+    rml:source "nullValues.json" ;
+    rml:referenceFormulation ql:JSONPath ;
+    rml:iterator "$" ;
+  ] ;
+  rr:subjectMap [
+    rr:template "http://example.com/Child/{$.Child[*].first}" ;
```

Comment:
Is there a practical difference between the notations `[*].first` and `.*.first`?

Reply:
No these are equivalent. There is some movement on standardization of JSONPath. See https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base and https://datatracker.ietf.org/doc/html/draft-ietf-jsonpath-base